src/plugins/elasticsearch/index.js

```diff
@@ -28,8 +28,8 @@ module.exports = function ({ Plugin }) {
           cert: string(),
           key: string()
         }).default(),
-        apiVersion: string().default('2.0'),
-        engineVersion: string().valid('^2.1.0').default('^2.1.0')
+        apiVersion: Joi.string().default('master'),
```

Comment:
How does the elasticsearch js client interpret `master` being passed as the API version? Wouldn't we still want to pin each version of Kibana to a specific version of the ES API?